src/integration_tests/regression_test.py

```diff
@@ -2,8 +2,8 @@
 
 from __future__ import annotations
 
-from collections.abc import Callable
 import math
+from collections.abc import Callable
```

Comment:
There are some changes about only the import order, which are not related to the essential point of this pull request. Could you revert these changes to prevent breaking the blame history?

(Btw, I think I need to introduce some static checker (e.g., isort) to constrain the style around imports.)

Reply:
No problem.

Reply:
Done